### ISSUE TITLE
SassC::SyntaxError in Users#index 

### ISSUE THREAD

**Issue by stefannibrasil** [thoughtbot/suspenders#1116]
When following the steps shared in #1106, I encountered the following error:

![Screenshot 2022-12-14 at 17-12-40 Action Controller Exception caught](https://user-images.githubusercontent.com/10670581/207748998-1da80222-8fc5-43d9-9095-d9ab794d8836.png)

This is what I have in the console:

```
ActionView::Template::Error (Error: File to import not found or unreadable: normalize.css/normalize.
        on line 4:1 of app/assets/stylesheets/application.scss
>> @import "normalize.css/normalize";
```

**Comment by stevepolitodesign** [comment]
@stefannibrasil thank you for opening this. We are working on a solution in #1113

**Comment by stevepolitodesign** [comment]
Closed by #1113

**Comment by stevepolitodesign** [comment]
We can now use postcss to load the postcss-normalize plugin.
